api/src/oscc.c

```diff
@@ -1092,3 +1092,84 @@ oscc_result_t get_device_name( char * string, char * const name )
 
     return result;
 }
+
+oscc_result_t get_wheel_speed_right_rear(
+    struct can_frame const * const frame,
+    double * wheel_speed_right_rear)
+{
+    if((frame == NULL) || (wheel_speed_right_rear == NULL)) { return OSCC_ERROR; }
```

Comment:
If we're still following the OSCC style guide this wouldn't be compliant, but I won't block the PR on this issue.